packages/environment-check/src/types.ts

```diff
@@ -38,7 +38,7 @@ export enum Extensions {
 export interface Environment {
     developmentEnvironment: DevelopmentEnvironment;
     platform: NodeJS.Platform;
-    versions: NodeJS.ProcessVersions;
+    versions: { [id: string]: string };
```

Comment:
The return interface of the new way: 
```
node -p process.versions
```
should be equivalent to what we previously did, like 
```
node -e "console.log(process.versions)"
```
I understand that the new approach returns the versions for installed node and the previous approach returns it for the current process, which in case we execute from VSCode returns the versions from VSCode's node, but the return structure should be the same, correct?

Do we need to change the type here?

Reply:
Reverted